packaging/create-deb.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# Exit on error
+set -e
+
+HOME=`pwd`
+AWK=/usr/bin/awk
+HEAD=/usr/bin/head
+GIT=/usr/bin/git
+SORT=/usr/bin/sort
+DCH=/usr/bin/dch
+PR=/usr/bin/pr 
+SED=/bin/sed
+FMT=/usr/bin/fmt
+PR=/usr/bin/pr
+XARGS=/usr/bin/xargs
+CUT=/usr/bin/cut
+
+
+cd $HOME/targets
+TARGETS=(*)
+echo "Targets: $TARGETS"
+cd $HOME
+
+PKG=openhim-core-js
+
+echo -n "Which version of the OpenHIM (from NPM) would you like this package to install? (eg. 1.3.0) "
+read OPENHIM_VERSION
```

Comment:
Maybe add a check in case it's left blank (or maybe grep out the current version in package.json as a default value?)